omero/users/cli/delete.rst

```diff
@@ -103,7 +103,7 @@ Including and excluding objects
 .. option:: --include
 
     Linked objects that would not ordinarily be deleted can be included in the
```

Comment:
Why is the text duplicating the option name?  This could simply be "Include linked objects that would not ordinarily be deleted".  The same applies to the redundancy in the other options.